src/app/pages/swap/components/swap-status/swap-status-item.tsx

```diff
@@ -0,0 +1,25 @@
+import { FiArrowUpRight } from 'react-icons/fi';
+
+import { Stack, Text } from '@stacks/ui';
+
+import { Flag } from '@app/components/layout/flag';
+import { SpaceBetween } from '@app/components/layout/space-between';
+
+interface SwapStatusItemProps {
+  icon: React.JSX.Element;
+  text: string;
+  timeStamp?: string;
```

Comment:
Maybe this `timeStamp` can be type Date?

Reply:
Timestamp is one word as well fyi

Reply:
That made me lol ...well, not too sure on what provides this to the UI so a bit unclear on type tbh. Pretty dumb still in the UI.